api/src/users/users.service.ts

```diff
@@ -157,7 +167,14 @@ export class UsersService {
     }
     if (!user)
       throw new NotFoundException();
-    return user;
+    let appUsers: AppUser[] = [];
+    user.forEach(user => {
+      appUsers.push({
+        ...user,
+        status: this.usersStatusGateway.getUserStatus(user?.id)
+      })
+    })
+    return appUsers;
```

Comment:
I think a `const usersWithStatus = users.map((u) => {...u, status: userStatus})` would be cleaner.

Reply:
Fix 1aada99